ElrondNetwork/elrond-go
# Implement redundancy for elrond nodes

* Implemented redundancy mechanism for nodes

**How to be tested:**

1. Start a normal system test and choose one node and with its validatorKey.pem start another two instances, 
one with flag **RedundancyLevel=1** and one with flag **RedundancyLevel=2** set in prefs.toml
3. After a while if we check the logs of those two nodes we should find in the node started with redundancy level set to 1 
the following info:
**"main or lower level redundancy machines are active   node redundancy level = 1"**
and in the node started with redundancy level set to 2 the following info:
**"main or lower level redundancy machines are active   node redundancy level = 2"**
4. After a while close the original instance and wait for a few minutes ~2. Now if we check the logs again we should find 
the following info:
**"main or lower level redundancy machines are inactive   node redundancy level = 1"** for the first node
**"main or lower level redundancy machines are active   node redundancy level = 2"** for the second node
5. After a while close the first redundancy node and after a few minutes ~2 check the logs of the second node and we should find the following info:
**"main or lower level redundancy machines are inactive   node redundancy level = 2"**
6. If after a while the original instance is restarted then after a few minutes ~2 we should notice the following info in the second node logs:
**"main or lower level redundancy machines are active   node redundancy level = 2"**
If we restarted both, the original instance and the first redundancy node, also in the logs of the first redundancy node we should find the following info:
**"main or lower level redundancy machines are active   node redundancy level = 1"**
